crud.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Product CRUD</title>
+  <link rel="stylesheet" href="/src/css/crud.css">
```

Comment:
In je HTML bestanden mis je nog belangrijke HTML meta tags: https://www.w3schools.com/css/css_rwd_viewport.asp Check ook de andere pagina's